L/LLVM@8/build_tarballs.jl

```diff
@@ -23,6 +23,10 @@ script = raw"""
 # We want to exit the program if errors occur.
 set -o errexit
 
+# arctic3 has about 40 cores limit ourselves to ~4 so that the 10 jobs can run in parallel
+# without stomping on each other
+nproc=4
```

Comment:
We're going to set `nproc` to be something like `10` for arctic.  I'm fine with super-over-subscription, but 10x is a little intense.  3-4x should be much better.  ;)